instrumentation/servlet/servlet-common/servlet-common.gradle

```diff
@@ -22,7 +17,4 @@ dependencies {
   }
   testImplementation group: 'org.eclipse.jetty', name: 'jetty-server', version: '7.0.0.v20091005'
   testImplementation group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '7.0.0.v20091005'
```

Comment:
i think?
```suggestion
  testLibrary group: 'org.eclipse.jetty', name: 'jetty-server', version: '7.0.0.v20091005'
  testLibrary group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '7.0.0.v20091005'
```

Reply:
Oops, no it wasn't - the previous `latestDepTest` was a ghost, it didn't specify the source set :P #1051